app/models/thought.rb

```diff
@@ -1,2 +1,4 @@
 class Thought < ApplicationRecord
+  belongs_to :user
```

Comment:
after out chat on Saturday about the DB architecture, is this still belongs_to or did we decide it's a different Rails active record association?

Reply:
This is a User - Thought relationship association that we did discussed :)  

One : Many relationship  
- User has many thoughts
- Thought belongs to User 

The link to diagram https://dbdiagram.io/d/5f9b12403a78976d7b79bc81